docs/source/theory/fktables.rst

```diff
@@ -0,0 +1,130 @@
+.. _fktables:
+
+============================================================
+Fast Interface (FK tables)
+============================================================
+
+.. raw:: latex
+
+   \maketitle
+
+.. raw:: latex
+
+   \tableofcontents
+
+Here we discuss the numerical implementation of the calculations of both the DIS structure functions
+and the hadronic cross-sections.
+
+In the framework of collinear QCD factorization, the :math:`F_2` structure function
+can be decomposed in terms of hard-scattering coefficient functions and PDFs as,
+
+.. math::
+
+    \begin{align} 
+    \label{eq:ev} 
```

Comment:
do we need the label? or the align?

Reply:
The labels no but actually the aligns I believe yes

Reply:
try - and resolve this comment